element/daterange/lang/en/customcertelement_daterange.php

```diff
@@ -33,7 +33,7 @@
 $string['dateranges'] = 'Dateranges';
 $string['fallbackstring'] = 'Fallback string';
 $string['fallbackstring_help'] = 'This string will be displayed if no daterange applied to a date. If Fallback string is not set, then there will be no output at all.';
-$string['help'] = 'Configure a string representation for each daterange. Set start and end dates as well as a string you would like to transform each range to. Make sure your ranges do not overlap, otherwise the first detected daterange will be applied. If no daterange applied to a date, then Fallback string will be displayed. If Fallback string is not set, then there will be no output at all. If you mark a date range as Recurring, then the configured year will not be considerred and the current year will be used. Also there is {{current_year}} placeholder that could  be used in the string representation. The placeholder will be replaced by the current year value in the certificate.';
+$string['help'] = 'Configure a string representation for each daterange. Set start and end dates as well as a string you would like to transform each range to. Make sure your ranges do not overlap, otherwise the first detected daterange will be applied. If no daterange applied to a date, then Fallback string will be displayed. If Fallback string is not set, then there will be no output at all. If you mark a date range as Recurring, then the configured year will not be considerred and the current year will be used. Also there are {{first_year}} and {{last_year}} and {{current_year}} placeholders that could  be used in the string representation. The placeholders will be replaced by first year or last year in the matched range or the current year.';
```

Comment:
There's a typo: "If no daterange _applies_ to a date, then Fallback string will be displayed"

And there's one more typo: "If you mark a date range as Recurring, then the configured year will not be _considered_ and the current year will be used."
After this sentence you should add: "As the year of a recurring date range is not considered, you are not allowed to configure a recurring date range with more than 12 months as it would become  ambiguous otherwise".